src/mode/modeNormal.ts

```diff
@@ -26,7 +26,7 @@ export default class NormalMode extends Mode {
         "dd" : () => { return vscode.commands.executeCommand("editor.action.deleteLines"); },
         "dw" : () => { return vscode.commands.executeCommand("deleteWordRight"); },
         "db" : () => { return vscode.commands.executeCommand("deleteWordLeft"); },
-        // "x" : () => { this.CommandDelete(1); }
+        "x" : () => { return vscode.commands.executeCommand("deleteRight"); },

```

Comment:
Tbh, this will likely get refactored in the future to better handle things like `4x` hence the CommandDelete implementation.